recipes/libopusenc/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+./configure --prefix=${PREFIX} --enable-static --enable-shared
```

Comment:
Please remove enable-static.  We prefer not to ship static binaries, and prefer to link instead. Thanks!